EIPS/eip-5131.md

```diff
@@ -0,0 +1,241 @@
+---
+eip: 5131
+title: ENS Subdomain Authentication
+description: Use ENS to link hot wallets as approved signers for a root ENS address.
```

Comment:
Using ENS subdomains to facilitate safer and more convenient signing operations.